src/BrowscapUpdater.php

```diff
@@ -154,6 +154,20 @@ public function getLogger()
     }
 
     /**
+     * Sets the Connection Timeout
+     *
+     * @param int $connectTimeout
+     *
+     * @return \BrowscapPHP\BrowscapUpdater
+     */
+    public function setConnectTimeout($connectTimeout)
+    {
+        $this->connectTimeout = (int) $connectTimeout;
+
+        return $this;
```

Comment:
Is it absolutely necessary to return `$this`? See: https://ocramius.github.io/blog/fluent-interfaces-are-evil/